releasenotes-dca/notes/manifest-collection-ga-41ad6d179ed789b5.yaml

```diff
@@ -0,0 +1,11 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+features:
+  - |
+    Enable orchestrator manifest collection by default
```

Comment:
maybe we should also add it to the main agent release note as it impacts both

Reply:
Added it 👍